locales/it/messages.json

```diff
@@ -2081,7 +2081,7 @@
     "receiverButtonBindMessage": {
         "message": "Richiesta di collegamento inviata al controllore di volo."
     },
-    "receiverButtonPassphrase": {
+    "receiverButtonBindingPhrase": {
         "message": "Password"
     },
```

Comment:
You are breaking the translations. You are changing the "key" of the translation, that will let the translation unused.
Simply don't touch the messages files (except the en language, of course) and they will be removed automatically in Crowdin.

Reply:
Ok, removed all changes to locales except for `locales/en/messages.json`.